app/controllers/panoptic/jobs/retries_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Panoptic
+  class Jobs::RetriesController < ApplicationController
+    def update
+      @job = Panoptic::FailedJob.find(params[:id])
+
+      @job.retry
+      redirect_to failed_jobs_path, notice: "Successfully retried job ##{@job.id}"
+
+    rescue => error
+      redirect_back fallback_location: failed_jobs_path,
+        alert: "Could not retry job - #{error.message.truncate(300)}"
+    end
```

Comment:
`SolidQueue::Job#retry` has the same result as `SolidQueue::Jobenqueue`, it prepares the job for execution. The job will not fail in this context but rather during its execution. 

The only problem that could arise here is if the job has already been retried successfully, which means that its failed_execution entry does not exist anymore. I have a suggestion for this below in `FailedJob`. 

As a result, I _think_ we can remove this `rescue` block